frontend/public/components/utils/datetime.ts

```diff
@@ -83,15 +83,9 @@ export const parsePrometheusDuration = (duration: string): number => {
 
 const zeroPad = (number: number) => (number < 10 ? `0${number}` : number);
 
-export const twentyFourHourTime = (date: Date): string => {
-  const hours = zeroPad(date.getHours());
-  const minutes = zeroPad(date.getMinutes());
-  return `${hours}:${minutes}`;
-};
-
-export const twentyFourHourTimeWithSeconds = (date: Date): string => {
-  const hours = zeroPad(date.getHours());
-  const minutes = zeroPad(date.getMinutes());
-  const seconds = zeroPad(date.getSeconds());
-  return `${hours}:${minutes}:${seconds}`;
+export const twentyFourHourTime = (date: Date, showSeconds?: boolean): string => {
```

Comment:
IMO, letting an undefined optional parameter evaluate to false when undefined is okay.  We have been leaning more towards optional args + optional chaining or null coalescence where needed. Mainly because optional chaining and null coalescence catch `null` **and** `undefined`, where  default parameter assignment only works for `undefined`.